### PR TITLE
Polyhedron_Demo : Documentation update

### PR DESCRIPTION
**Pull request by maxGimeno**
- Documents the shader programs and explains in what situation they must be used.

### CODE REVIEW

**Comment by sloriot** [comment]
Merged in CGAL-4.8-Ic-117

File: Three/include/CGAL/Three/Viewer_interface.h [hunk 1]
```diff
@@ -43,6 +43,31 @@ class VIEWER_EXPORT Viewer_interface : public QGLViewer, public QOpenGLFunctions
...
+   * PROGRAM_INSTANCED_WIRE :
+   * used to display instanced rendered wired spheres. Not affected by light.
+   * PROGRAM_C3T3 :
+   * used to render a c3t3_item. It discards any fragment on a side of a plane, meaning that nothing is displayed on this side of the plane. Affected by light.
+   * PROGRAM_C3T3_EDGES,
+   * used to render the edges of a c3t3_item. It discards any fragment on a side of a plane, meaning that nothing is displayed on this side of the plane. Not affected by light.
+   * NB_OF_PROGRAMS :
+   * holds the number of different programs in this enum.
+   */
   enum OpenGL_program_IDs { PROGRAM_WITH_LIGHT,
```
Comment (lrineau) [inline Three/include/CGAL/Three/Viewer_interface.h]: Is not there a better way to document enumators?

@sloriot?

**Comment by afabri** [comment]
See the [doxygen manual page for `enum`](http://www.stack.nl/~dimitri/doxygen/manual/docblocks.html#memberdoc)

**Comment by maxGimeno** [comment]
Done.

**Comment by sloriot** [comment]
Merged in CGAL-4.8-Ic-121

**Comment by sloriot** [comment]
Tested locally

**Pull request merged**